app/src/marketplace/components/MarkdownEditor/index.jsx

```diff
@@ -31,9 +31,9 @@ const MarkdownEditor = ({ value, placeholder, onChange, className }: Props) => {
     }, [text])
 
     return (
-        <div className={styles.root}>
+        <div className={cx(styles.root, className)}>
```

Comment:
Good, yeah should really avoid passed className being set on anything but the top level element, very annoying if you can't control class nesting.